.github/workflows/deploy-prod.yml

```diff
@@ -1,6 +1,9 @@
-name: Deploy to Prod
+name: Deploy to Prod (ECS)
 
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+  push:
```

Comment:
For prod, we actually *don't* want this. Basically, your workflows here should maintain the previous behavior. I had you add the `push: branches: [master]` bit to the dev one because the old dev one had it.

This `on:` section says when the workflow should run. `on: workflow_dispatch` means the workflow should run when we manually trigger it via the GitHub interface, going to "Actions", and clicking the "Deploy to Prod". `on: push: branches: [master]` means it *automatically* run when new code is added to the master branch (i.e.: a PR is merged). We want that for the dev environment, but not for prod; for prod we only deploy when we're around to monitor for it, and after searching the dev logs a bit to make sure we haven't introduced any issues.

Reply:
Complete 👍